components/camel-kubernetes/src/main/java/org/apache/camel/component/kubernetes/KubernetesConfiguration.java

```diff
@@ -105,6 +105,21 @@
     @UriParam(label = "advanced")
     private Integer connectionTimeout;
 
+    @UriParam(label = "consumer")
+    private String cRDName;
+
+    @UriParam(label = "consumer")
+    private String cRDGroup;
```

Comment:
Please use crdGroup cRDGroup is really ugly to read and see :-)

Reply:
I change.